diff --git a/qpid/extras/qmf/setup.py b/qpid/extras/qmf/setup.py
index c7626aa..97ff0f5 100755
--- a/qpid/extras/qmf/setup.py
+++ b/qpid/extras/qmf/setup.py
@@ -23,7 +23,7 @@ setup(name="qpid-qmf",
       version="0.7",
       author="Apache Qpid",
       author_email="dev@qpid.apache.org",
-      packages=["qmf", "qmf2", "qmf2.tests"],
+      packages=["qmf"],
       package_dir={"": "src/py"},
       url="http://qpid.apache.org/",
       license="Apache Software License",
